### PR TITLE
Laravel 6.0 Shift

### PR DESCRIPTION
**Pull request by freekmurze**
This pull request includes the changes for upgrading to Laravel 6.0. Feel free to commit any additional changes to the `shift-17748` branch.

**Before merging**, you should:

- Checkout the `shift-17748` branch
- Review **all** pull request comments for additional changes
- Update your dependencies for Laravel 6.0
- Run `composer update` (if the scripts fail, add `--no-scripts`)
- Thoroughly test your application ([no tests?](https://confidentlaravel.com))

If you want help with your upgrade, check out the [Shift Human Services](https://laravelshift.com/human-services) or join the [Shifty Coders](https://laravelshift.com/shifty-coders) Slack room to get answers to all your Laravel questions.

### CODE REVIEW

**Comment by freekmurze** [comment]
:warning: Shift defaulted the following configuration files. This was done in the [Default config files](https://github.com/spatie/dashboard.spatie.be/pull/151/commits/6247d70cc6d11eb9fbc3d9cc30f95497f606895f?w=1) commit so you may easily review and backfill any of your customizations.

- [ ] config/broadcasting.php
- [ ] config/database.php
- [ ] config/queue.php
- [ ] config/services.php

**Comment by freekmurze** [comment]
:information_source: The Eloquent model's `toArray()` method will now cast any attributes that implement `Illuminate\Contracts\Support\Arrayable` to an array. While unlikely to impact your application, this change could effect code which relies upon the previous data type or output.

**Comment by freekmurze** [comment]
:information_source: Laravel 6.0 made [performance optimizations](https://github.com/laravel/framework/pull/28153) for _integer_ key types. If you are using a _string_ as your model's primary key, you may set the `$keyType` property on your model.

```php
/**
 * The "type" of the primary key ID.
 *
 * @var string
 */
protected $keyType = 'string';
```

**Comment by freekmurze** [comment]
:information_source: The `mandrill` and `sparkpost` mail drivers, as well as the `rackspace` storage driver were removed in Laravel 6.0. If you were using these drivers, you may adopt a community maintained package which provides the driver.

**Comment by freekmurze** [comment]
:information_source: Previous versions of Laravel would retry jobs indefinitely. Beginning with Laravel 6.0, the `php artisan queue:work` now tries a job **one time by default**. If you want to force jobs to be tried indefinitely, you may pass the `--tries=0` option.

**Comment by freekmurze** [comment]
:information_source: Shift noticed you have additional namespaces in your application. You should consider using the [Consolidate Namespaces Shift](https://laravelshift.com/laravel-consolidate-custom-namespaces) to simplify your namespaces into the default Laravel `App` namespace.

**Comment by freekmurze** [comment]
:alembic: This Shift is still being refined. **Please report any issues or suggestions**. Your feedback is what helps improve the experience for everyone.

**Comment by freekmurze** [comment]
:tada: Congratulations, you're now running the latest version of Laravel.

The following _Shifts_ can ensure your app is fully upgraded and following the practices recommended by Laravel and the community:
- [Laravel Linter](https://laravelshift.com/opinionated-laravel-way-shift) analyzes your codebase for potential opportunities to do things "The Laravel Way".
- [Laravel Fixer](https://laravelshift.com/laravel-code-fixer) automates many of the suggestions found by the _Linter_ and other _Shifts_.
- [Upgrade Checker](https://laravelshift.com/laravel-upgrade-checker) ensures your application is fully upgraded by detecting any outdated code.

In addition, stay upgraded forever as well as run these _Shifts_ by subscribing to one of the new [Shifty Plans](https://laravelshift.com/shifty-plans).

File: app/Http/Kernel.php [hunk 1]
```diff
@@ -39,6 +39,7 @@ class Kernel extends HttpKernel
      */
     protected $routeMiddleware = [
         'auth' => \App\Http\Middleware\Authenticate::class,
+        \Illuminate\Routing\Middleware\ThrottleRequests::class,
```
Comment (sebastiandedeyne) [inline app/Http/Kernel.php]: Looks like the `'throttle'` key is missing 👀 @jasonmccreary

Comment (jasonmccreary) [reply app/Http/Kernel.php]: That's not exactly it, it actually [belongs in the priority group](https://github.com/laravel/laravel/blob/master/app/Http/Kernel.php#L76). That's where it should have been added. I'll tweak the hooks to ensure it skips this part.

**Pull request merged**